.eslintrc.js

```diff
@@ -32,5 +32,6 @@ module.exports = {
     }],
     'no-undef': 'off',
     'no-var': 'error',
+    eqeqeq: 'error',
```

Comment:
This is one of the lint rules that can catch ambiguous and error prone comparisons like:
`'' == 0` (true)